src/Components/Nav/Components/LoginModal.js

```diff
@@ -1,33 +1,69 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
-import mixin from "../../../Styles/mixin";
-import NavSvg from "../NavSvg";
+import { useHistory } from "react-router";
 import {
   ModalContainer,
   ModalWrapper,
   ModalHeader,
   ModalContent,
   FullWideBtn,
   OrBorder,
+  InfoSection,
+  InfoInput,
   ExtraAction,
   ExtraSection,
 } from "./ReusableStyle";
+import { signInAPI } from "../../../config";
+import mixin from "../../../Styles/mixin";
+import NavSvg from "../NavSvg";
 
-export default function LoginModal({ handleClose }) {
+export default function LoginModal({ handleLogin }) {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
 
+  const history = useHistory();
+
   const handleLoginBtn = (e) => {
-    console.log("checked");
+    var exptext = /^[A-Za-z0-9_]+@[A-Za-z0-9]+\.[A-Za-z0-9]+/;
+    if (!email) return alert("이메일이 필요합니다.");
+    else if (!exptext.test(email)) return alert("이메일을 입력하세요.");
+    else if (!password) return alert("비밀번호를 입력하세요.");
+    else if (password.length < 8)
+      return alert(
+        "비밀번호는 최소 8자 이상이어야 합니다. 다시 시도해 주세요."
+      );
+
+    if (exptext.test(email) && password.length >= 8) {
+      fetch(`${signInAPI}/user/signin`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          email: email,
+          password: password,
+        }),
+      })
+        .then((res) => res.json())
+        .then((res) => {
+          if (res.access_token) {
+            alert("로그인에 성공했습니다.");
+            sessionStorage.setItem("access_token", res.access_token);
+            history.push("/");
+          } else {
+            alert("이메일 및 비밀번호를 확인하세요.");
+          }
+        });
+    }
   };
 
   return (
     <ModalContainer>
       <ModalWrapper>
         <ModalHeader>
-          <div onClick={handleClose}>{NavSvg.modalClose}</div>
+          <div onClick={() => handleLogin()}>{NavSvg.modalClose}</div>
```

Comment:
```suggestion
          <div onClick={handleLogin}>{NavSvg.modalClose}</div>
```
함수에 전달하는 인자가 없으므로 즉시실행 함수로 만들어줄 필요 없습니다